packages/medusa-plugin-algolia/package.json

```diff
@@ -1,46 +1,32 @@
 {
   "name": "medusa-plugin-algolia",
   "version": "0.2.9",
-  "description": "Search support for algolia",
-  "main": "index.js",
+  "description": "Algolia search plugin for Medusa",
   "repository": {
     "type": "git",
     "url": "https://github.com/medusajs/medusa",
     "directory": "packages/medusa-plugin-algolia"
   },
-  "author": "rolwin100",
+  "files": [
+    "dist"
+  ],
+  "author": "Medusa",
   "license": "MIT",
   "scripts": {
     "prepare": "cross-env NODE_ENV=production yarn run build",
     "test": "jest --passWithNoTests src",
-    "build": "babel src --out-dir . --ignore '**/__tests__','**/__mocks__'",
-    "watch": "babel -w src --out-dir . --ignore '**/__tests__','**/__mocks__'"
-  },
-  "peerDependencies": {
-    "medusa-interfaces": "1.3.6",
-    "typeorm": "0.x"
+    "build": "tsc",
+    "watch": "tsc --watch"
   },
   "dependencies": {
-    "algoliasearch": "^4.10.5",
-    "body-parser": "^1.19.0",
-    "lodash": "^4.17.21",
-    "medusa-core-utils": "^1.1.39",
-    "medusa-interfaces": "^1.3.6"
+    "@medusajs/types": "*",
```

Comment:
todo: move to devDeps, types does not exists once it is build. They are just of use for the development

Reply:
@olivermrbl any particular reason why it is resolved but not done?